plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraSplitManager.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.cassandra;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.connector.ColumnHandle;
+import io.trino.spi.connector.ConnectorSplit;
+import io.trino.spi.connector.ConnectorSplitSource;
+import io.trino.spi.predicate.NullableValue;
+import io.trino.spi.predicate.TupleDomain;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.parallel.Execution;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static io.trino.plugin.cassandra.CassandraTestingUtils.CASSANDRA_TYPE_MANAGER;
+import static io.trino.plugin.cassandra.CassandraTestingUtils.createKeyspace;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
+import static org.junit.jupiter.api.parallel.ExecutionMode.CONCURRENT;
+
+@TestInstance(PER_CLASS)
+@Execution(CONCURRENT)
+public class TestCassandraSplitManager
+{
+    private static final String KEYSPACE = "test_cassandra_split_manager_keyspace";
+
+    private CassandraServer server;
+    private CassandraSession session;
+
+    @BeforeAll
+    public void setUp()
+            throws Exception
+    {
+        server = new CassandraServer();
+        session = server.getSession();
+        createKeyspace(session, KEYSPACE);
+    }
+
+    @AfterAll
+    public void tearDown()
+    {
+        server.close();
+        server = null;
+        session.close();
+        session = null;
+    }
+
+    @Test
+    public void testGetSplitsWithSinglePartitionKeyColumn()
```

Comment:
should we move this test to TestCassandraConnectorTest using tpch keyspace?

Reply:
I don't think we should. I'm afraid that this test is too trivial for TestCassandraConnectorTest to deal with. If it becomes responsible for this test case, it will have more and more test cases in the future.